ow_faults_detection/CMakeLists.txt

```diff
@@ -0,0 +1,212 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(ow_faults_detection)
+
+set(EXEC_NAME faults_detection)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED COMPONENTS
+  dynamic_reconfigure
+  roscpp
+  rospy
+  ow_lander
+  std_msgs
+  message_generation
+  ow_faults_injection
```

Comment:
@Samahu I understand the dependency.  It's there because ow_faults_injection defines the messages used here.  But I agree the naming is confusing and misleading.

Reply:
I would suggest all fault messages to be placed under **ow_fault** namesapce/package.

Reply:
Since @lanseafood plans to move the messages to ow_faults_detection, I think this solves all the issues and I retract my two suggestions above.